README.md

```diff
@@ -229,6 +231,21 @@ cfg_cmds.each do |cmd|
 end
 ```
 
+```ruby
+# Add static machine key at site level
+iis_config "MySite /commit:site /section:machineKey /validation:AES /validationKey:AAAAAA /decryptionKey:ZZZZZ" do
+  action :config
+end
```

Comment:
Agreed, this should be `:set` as that is the new command.  Or simply no action at all since it is also the default.